lib-ee/emqx_license/test/emqx_license_test_lib.erl

```diff
@@ -47,6 +47,32 @@ test_key(Filename, Format) ->
             public_key:pem_entry_decode(PemEntry)
     end.
 
+make_license(Values0 = #{}) ->
+    Defaults = #{
+        license_format => "220111",
+        license_type => "0",
+        customer_type => "10",
+        name => "Foo",
+        email => "contact@foo.com",
+        deployment => "bar-deployment",
+        start_date => "20220111",
+        days => "100000",
+        max_connections => "10"
+    },
+    Values1 = maps:merge(Defaults, Values0),
+    Keys = [
```

Comment:
nit: `maps:keys(Defaults)`?

Reply:
In this case, I used a list to make the key order deterministic to match the previous `make_license/1` behavior.